backend/tests/test_participants_list.py

```diff
@@ -46,9 +46,11 @@ def test_create_participant_when_logged_in(app, client, access_token, new_partic
         participant = schema.dump(Participant.query.first())
 
     assert rv.status_code == HTTPStatus.CREATED
-    assert response["participant"]["email"] == new_participant["email"]
+    assert response["email"] == new_participant["email"]
     for value in new_participant.values():
         assert value in participant.values()
+    for value in [*participant.keys()]:
```

Comment:
```suggestion
    for value in participant.keys():
```

Reply:
Hmm, I don't even recall, why I did this in such manner. Nevertheless, changed.